package/upgrade/upgrade_node.sh

```diff
@@ -370,6 +374,8 @@ EOF
 }
 
 command_pre_drain() {
+  recover_rancher_system_agent
```

Comment:
pre_drain only works for 2+ node. We need to handle the single-node case as well.